test.py

```diff
@@ -363,6 +363,16 @@ def test_prefix_set(self):
         self.assertEqual([long_key], list(ps.iter(self._LONG_KEY)))
         self.assertEqual([other_key], list(ps.iter(self._OTHER_KEY)))
 
+    def testStringTrieSeparator(self):
```

Comment:
test_string_trie_separator

Furthermore, this should be defined in StringTrieTestCase.